package.json

```diff
@@ -17,7 +17,7 @@
     "build:app": "webpack --progress --mode production",
     "check-types": "tsc --sourceMap false --noEmit",
     "dev": "run-p dev:*",
-    "dev:css": "postcss src/tailwind.css -o src/index.css --watch",
+    "dev:css": "npx tailwindcss -i ./src/tailwind.css -o ./src/index.css --watch",
```

Comment:
hmm, whats the diff?
i think under the hood tailwindcss cli is also using postcss?

Reply:
when using the postcss the styles are not updating, the JIT compiler not working. thats y i change to tailwindcss cli.

Reply:
ah okok